### ISSUE TITLE
adopt a process for reconciling books

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#367]
Probably on a monthly basis, once they're reconciled initially.

**Comment by whit537** [comment]
Looking at this with @kaguillera. We're thinking of this in terms of improvements to:

http://inside.gratipay.com/appendices/finances

**Comment by whit537** [comment]
PR started in #390.

**Comment by techtonik** [comment]
What is this "book reconciliation" in layman terms? (looking at https://github.com/gratipay/inside.gratipay.com/pull/418)

**Comment by whit537** [comment]
Closing in favor of #390.

**Comment by whit537** [comment]
@techtonik Reconciling books means comparing cross-checking various accounting systems, in our case, these three:

 - payment processor statements
 - our bank statements
 - the Gratipay.com database
